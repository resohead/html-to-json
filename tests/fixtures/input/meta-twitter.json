[
    { "charset": "utf-8" },
    {
        "name": "viewport",
        "content": "width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0"
    },
    { "http-equiv": "x-ua-compatible", "content": "IE=edge,chrome=1" },
    { "http-equiv": "X-UA-Compatible", "content": "IE=11" },
    { "name": "twitter:card", "content": "twitter-card" },
    { "name": "twitter:site", "content": "@organisation" },
    { "name": "twitter:title", "content": "Example Site Twitter Meta" },
    {
        "name": "twitter:description",
        "content": "This is a test og twitter description tag"
    },
    { "name": "twitter:creator", "content": "@handle" },
    {
        "name": "twitter:image",
        "content": "https://example.com/twitter-social-image.jpg"
    }
]
