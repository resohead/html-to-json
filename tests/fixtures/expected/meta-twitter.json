{
    "hybrid": {
        "url": null,
        "handle": "@organisation",
        "title": "Example Site Twitter Meta",
        "name": null,
        "description": "This is a test og twitter description tag",
        "type": "website",
        "images": [
            "https://example.com/twitter-social-image.jpg"
        ],
        "image": "https://example.com/twitter-social-image.jpg",
        "author": "@handle",
        "authorHandle": "@handle",
        "keywords": null
    },
    "openGraph": {
        "images": []
    },
    "twitter": {
        "images": ["https://example.com/twitter-social-image.jpg"],
        "card": "twitter-card",
        "site": "@organisation",
        "title": "Example Site Twitter Meta",
        "description": "This is a test og twitter description tag",
        "creator": "@handle",
        "image": "https://example.com/twitter-social-image.jpg"
    }
}
